### PR TITLE
 Add ci config - .travis.yml

### PR DESCRIPTION
**Pull request by dbalinov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
```
Comment (ventsislav-georgiev) [inline .travis.yml]: If we use a preview version for the CI builds and an official release version for development, we may encounter issues with incompatibilities. Is there no official version supported here?

Comment (dbalinov) [reply .travis.yml]: Travis supports only this two versions:
1.0.0-preview2-003121   Microsoft .NET Core 1.0.0 - SDK Preview 2
1.0.0-preview2-003131   Microsoft .NET Core 1.0.1 - SDK 1.0.0 Preview 2

I can update to .NET Core 1.0.1

Comment (ventsislav-georgiev) [reply .travis.yml]: The latest version seems to be 1.0.2 - https://github.com/dotnet/core/releases
IMO we better use the closest supported version e.g. 1.0.1

**Review by ventsislav-georgiev — APPROVED** [review]
Consider using .NET Core 1.0.1

**Pull request merged**